### DISCUSSION TITLE
Vectorized casos.PS.sym

### DISCUSSION THREAD

**Discussion by JOlucak** [iFR-ACSO/casos#33] [Q&A] [answered]
Short problem description:

We want to generate a vector of polynomials which is basically a copy lam_g of other polynomial sos.g

for k = 1:length(sos.g)
      lam_g(k) = casos.PS.sym(['lam_g' num2str(k)],basis(sos.g(k)));
end

Question: Is it possible to avoid this for loop e.g.  lam_g = casos.PS.sym(names ,basis(sos.g));
 
where basis(sos.g) returns a matrix of the monomials from the original polynomial and names is a char array containing names of the newly generate polynomials.

**Comment by tcunis** [comment, marked as answer]
The suggested syntax is added by commit [d55260e](https://github.com/iFR-ACSO/casos/commit/d55260ece8c6892b0cb8c484109d9a9f57813b03).
